openstack/neutron/templates/asr_config_agents/_deployment-asr1k.yaml.tpl

```diff
@@ -41,8 +41,13 @@ spec:
         - name: neutron-asr1k
           image: {{$context.Values.global.registry}}/loci-neutron:{{$context.Values.imageVersionASR1k | default $context.Values.imageVersion | required "Please set neutron.imageVersionASR1k or similar"}}
           imagePullPolicy: IfNotPresent
-          command:
-            - /container.init/neutron-asr1k-start
+          command: ['/var/lib/openstack/bin/python']
```

Comment:
any reason to hardcode python path instead using the exectuable asr1k-l3-agent which has a python shebang?

Reply:
Honestly, my tooling for debugging (skaffold) recognizes a python interpreter path in the deployment and replaces it with a call to the debugger. If I call the executable, that mechanism won't work anymore.

Reply:
Valid enough for me, fine.